src/config.rs

```diff
@@ -703,6 +703,16 @@ impl TiKvConfig {
             return Err("default rocksdb not exist, buf raftdb exist".into());
         }
 
+        if self.raft_store.range_deletion_delay.as_secs()
+            < self.server.end_point_request_max_handle_secs + 60
```

Comment:
Can we use range_deletion_delay as end_point_request_max_handle_secs + 60 directly?